pl/cheatsheets/index.md

```diff
@@ -0,0 +1,86 @@
+|  `1 :: List(2,3)`                                                                                        |  operator dołożenia elementu na początek listy. |
+|  `1 to 5` _to samo co_ `1 until 6` <br> `1 to 10 by 2`                                                      |  składnia dla przedziałów. |
+|  `()` _(puste nawiasy)_                                                                                   |  jedyny obiekt typu Unit (podobnie jak void w C/Java). |
+|  <span id="control_constructs" class="h2">konstrukcje kontrolne</span>                                                     |                 |
+|  `if (check) happy else sad`                                                                             |  warunek. |
+|  `if (check) happy` _to samo co_ <br> `if (check) happy else ()`                                            |  lukier składniowy dla warunku. |
+|  `while (x < 5) { println(x); x += 1}`                                                                   |  pętla while. |
+|  `do { println(x); x += 1} while (x < 5)`                                                                |  pętla do while. |
+|  `import scala.util.control.Breaks._`<br>`breakable {`<br>`    for (x <- xs) {`<br>`        if (Math.random < 0.1) break`<br>`    }`<br>`}`|  instrukcja przerwania pętli (break). ([slides](http://www.slideshare.net/Odersky/fosdem-2009-1013261/21)) |
+|  `for (x <- xs if x%2 == 0) yield x*10` _to samo co_ <br>`xs.filter(_%2 == 0).map(_*10)`                    |  zrozumienie pętli for: filtrowanie / mapowanie |
+|  `for ((x,y) <- xs zip ys) yield x*y` _to samo co_ <br>`(xs zip ys) map { case (x,y) => x*y }`              |  zrozumienie pętli for: przypisanie z podziałem |
+|  `for (x <- xs; y <- ys) yield x*y` _to samo co_ <br>`xs flatMap {x => ys map {y => x*y}}`                  |  zrozumienie pętli for: iloczyn kartezjański |
+|  `for (x <- xs; y <- ys) {`<br>    `println("%d/%d = %.1f".format(x,y, x*y))`<br>`}`                     |  zrozumienie pętli for: imperatywnie<br>[sprintf-style](http://java.sun.com/javase/6/docs/api/java/util/Formatter.html#syntax) |
+|  `for (i <- 1 to 5) {`<br>    `println(i)`<br>`}`                                                        |  zrozumienie pętli for: iterowanie aż do górnej granicy |
+|  `for (i <- 1 until 5) {`<br>    `println(i)`<br>`}`                                                     |  zrozumienie pętli for: iterowanie poniżej górnej granicy |
+|  <span id="pattern_matching" class="h2">pattern matching (dopasowywanie wzorca)</span>                                                         |                 |
+|  <span class="label success">Dobrze</span> `(xs zip ys) map { case (x,y) => x*y }`<br> <span class="label important">Źle</span> `(xs zip ys) map( (x,y) => x*y )` |  używaj słowa kluczowego case w funkcjach w celu dopasowywania wzorca. |
```

Comment:
I'm not sure about this suggestion. Do you want me to replace something with `(pattern matching)`? I've added that phrase at the end.